packages/webdriverio/src/commands/element/click.js

```diff
@@ -37,11 +37,40 @@
  * </example>
  *
  * @alias element.click
- * @uses protocol/element, protocol/elementIdClick
+ * @uses protocol/element, protocol/elementIdClick, protocol/performActions, protocol/positionClick
  * @type action
- *
+ * @param {Object=} options object containing a property called `button` which can be set to 0 (left) 1 (middle) or 2 (right) button
  */
 
-export default function click () {
-    return this.elementClick(this.elementId)
+export default async function click (options) {
+    const { button } = options || {}
+    const isLeftButton = button === 'left' || button === 0 || !button
+    const isMiddleButton = button === 'middle' || button === 1
+    const isRightButton = button === 'right' || button === 2
+
+    if (isLeftButton) {
```

Comment:
I'd use perform actions if user passed button in any case. There might be a case when element is not clickable and perform actions will do the trick

Reply:
so we don't want to use the `elementClick` command at all?

Reply:
Alright, I can see that being useful. I'll push the change when I can.

Reply:
makes sense to me